src/Cms/AppPlugins.php

```diff
@@ -225,6 +228,57 @@ protected function extendFileMethods(array $methods): array
         return $this->extensions['fileMethods'] = File::$methods = array_merge(File::$methods, $methods);
     }
 
+    /**
+     * Registers additional custom file types and mimes
+     *
+     * @param array $fileTypes
+     * @return array
+     */
+    protected function extendFileTypes(array $fileTypes): array
+    {
+        // normalize array
+        foreach ($fileTypes as $ext => $file) {
+            $extension = $file['extension'] ?? $ext;
+            $type      = $file['type'] ?? null;
+            $mime      = $file['mime'] ?? null;
+            $resizable = $file['resizable'] ?? false;
+            $viewable  = $file['viewable'] ?? false;
+
+            if ($type !== null) {
+                if (isset(F::$types[$type]) !== true) {
+                    F::$types[$type] = [];
+                }
+
+                if (in_array($extension, F::$types[$type]) === false) {
+                    F::$types[$type][] = $extension;
+                }
+            }
+
+            if ($mime !== null) {
```

Comment:
```suggestion
            if (is_string($mime) === true) {
```

Reply:
@lukasbestle I had to revert because of mime can be string or array.